restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/BaseVoiceInterpreter.java

```diff
@@ -1517,11 +1531,42 @@ public Gathering(final ActorRef source) {
 
         @Override
         public void execute(final Object message) throws Exception {
+
+            // check mandatory attribute partialResultCallback
+            Attribute resultCallbackAttr = verb.attribute(GatherAttributes.ATTRIBUTE_PARTIAL_RESULT_CALLBACK);
             final NotificationsDao notifications = storage.getNotificationsDao();
+            if (resultCallbackAttr == null) {
```

Comment:
@gvagenas @YevgenL  we should stop it for `SPEECH` and `DTMF_SPEECH`
so let's add statement `inputType != DTMF` too